### ISSUE TITLE
Bug for "inherit" when font-weight-notation is "numeric"

### ISSUE THREAD

**Issue by smutnyleszek** [stylelint/stylelint#934]
When `font-weight-notation` is set to `numeric` I would expect it to still allow for `inherit`.

```css
.foo {
    font-weight: inherit;
}

.bar {
    font-weight: 400;
}
```

```json
{
    "rules": {
        "font-weight-notation": ["numeric", {
            "ignore": ["relative"],
            "severity": "warning"
        }]
    }
}
```

I use stylelint 5.0.1. It happens in Atom.

**Comment by davidtheclark** [comment]
@smutnyleszek : If you can file a PR with at least failing tests that illustrate the need, that would really help speed up this fix.

**Comment by davidtheclark** [comment]
Closed by #931.
